rpcserver.go

```diff
@@ -5938,19 +5931,14 @@ func (r *rpcServer) GetNodeInfo(ctx context.Context,
 
 	features := invoicesrpc.CreateRPCFeatures(node.Features)
 
-	return &lnrpc.NodeInfo{
-		Node: &lnrpc.LightningNode{
-			LastUpdate: uint32(node.LastUpdate.Unix()),
-			PubKey:     in.PubKey,
-			Addresses:  nodeAddrs,
-			Alias:      node.Alias,
-			Color:      routing.EncodeHexColor(node.Color),
-			Features:   features,
-		},
-		NumChannels:   numChannels,
-		TotalCapacity: int64(totalCapacity),
-		Channels:      channels,
-	}, nil
+	return &lnrpc.LightningNode{
```

Comment:
If we're touching this code, can we fix the pre-allocation above (`nodeAddrs`) since we know how many elements there will be?

Reply:
Fixed